source/manual/retiring-an-application.html.md

```diff
@@ -119,8 +119,13 @@ application should be removed.
 ## 12. Update DNS
 
 Request any public DNS entries be removed. If the app had an admin UI, it will
-have had public DNS entries in the `publishing.service.gov.uk` domain. Request
-that the infrastructure team remove these, or, if appropriate, transition them.
+have had public DNS entries in the `publishing.service.gov.uk` domain.
+
+Follow the instructions [here][dns-changes] in order to remove these, and ask
```

Comment:
I think `Follow the [instructions for DNS changes][dns-changes] in order to remove these` would be better semantically and for accessibility.